public/minimal.js

```diff
@@ -15,28 +17,52 @@ window.ready = (callback) => {
   }
 }
 
-async function connect() {
-  if (!_token) return
-  
-  client = await SignalWire.SignalWire({
-    host: _host,
+async function initSWClient() {
+  if (!_token) {
+    console.error('Token not found')
+    return
+  }
+
+  client = await SWire({
+    host: !!_host && _host.trim().length ? _host : undefined,
     token: _token,
-    rootElement: document.getElementById('rootElement'),
+    debug: {
+      logWsTraffic: true,
+    },
+    logLevel: 'debug',
   })
 
-  await client.connect()
+  return client
 }
 
-async function makeCall() {
+async function connect() {
```

Comment:
@giavac; with this method, you can set the token via browser console:

```ts
window._token = '.......'
```

And then call this `connect()` method.